controls/cis_ocp_1_4_0/section-1.yml

```diff
@@ -309,30 +309,36 @@ controls:
     controls:
     - id: 1.3.1
       title: Ensure that controller manager healthz endpoints are protected by RBAC
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - rbac_debug_role_protects_pprof
```

Comment:
Since the original 1.3.1 control was removed, could we fix the control references in the rules?